### PR TITLE
(MODULES-3340) Release 1.1.2

### PR DESCRIPTION
**Pull request by cyberious**
- CHANGELOG and metadata update

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,13 @@
...
+##2015-08-13 - SUpported Release 1.1.2
+###Summary
+
+Fix critical bug when writing dword and qword values 
+
+####Bugfixes
+- Fix the way we write dword and qword values
+- Change the way we convert to byte arrays
+
+##~~2015-08-12 - Supported Release 1.1.1~~ - Deleted
```
Comment (ferventcoder) [inline CHANGELOG.md]: @cyberious you might as well add in the MODULES-1921 fix in this deleted release.

Comment (cyberious) [inline CHANGELOG.md]: You asking to add the ticket or the details?

Comment (ferventcoder) [inline CHANGELOG.md]: the old release didn't mention the summary or details of what the fix for that was about.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +15,9 @@ This release adds Puppet Enterprise 2015.2.0 to metadata
 ####Features
 - Testcase fixes
 - Gemfile updates
+- Updated the logic to by which we convert to byte array
```
Comment (ferventcoder) [inline CHANGELOG.md]: wording.

**Pull request merged**